From 8fa75767359f78c5dc6a9ef263ff72ee94c6454c Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 26 Aug 2015 22:24:44 +0300
Subject: [PATCH 0361/1240] fix: comphy: fixed register set function in comphy
 init code

- Need to fix the Armada-38x Comphy init

Change-Id: Ibfda3a63e4b227ef335a09186cca5a01322b024d
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/23222
Tested-by: Star_Automation <star@marvell.com>
---
 drivers/phy/comphy_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/comphy_core.c b/drivers/phy/comphy_core.c
index d0d5da3..5b6e275 100644
--- a/drivers/phy/comphy_core.c
+++ b/drivers/phy/comphy_core.c
@@ -73,7 +73,7 @@ static char *get_type_string(enum phy_type type)
 	return type_strings[type];
 }
 
-void reg_set(void __iomem *addr, u32 mask, u32 data)
+void reg_set(void __iomem *addr, u32 data, u32 mask)
 {
 	u32 reg_data;
 	debug("Write to address = %#010lx, data = %#010x (mask = %#010x) - ", (unsigned long)addr, data, mask);
-- 
1.9.1

